webpack.config.js

```diff
@@ -20,9 +25,11 @@ module.exports = merge(baseConfig, extractCssConfig, {
   },
   plugins: [
     new HtmlWebpackPlugin({
+      alwaysWriteToDisk: true,
```

Comment:
this is for a better performance... or something else?

Reply:
We need to write the HTML file on disk, not just serve it. It's needed by the docker container which is already serving the file.